src/styles/blocks/card/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  margin-right: $card-margin;
+  padding: 32px 16px 16px;
+  border: 1px solid $color-smoke;
+  border-radius: 5px;
+
+  &__product-photo {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    padding-top: 40px;
+    font-size: $font-size-medium;
+    line-height: $line-height-big;
+    font-weight: 500;
+    color: $color-main;
+  }
+
+  &__product-code {
+    padding-top: 4px;
+    font-size: $font-size-small;
+    line-height: $line-height-small;
+    font-weight: $font-weight-normal;
```

Comment:
no need to set this explicitly. normal is normal even without setting it to normal.